manifests/vrrp/track_process.pp

```diff
@@ -20,12 +20,17 @@
 # $full_command::  Match entire process cmdline
 #             Default: undef
 #
+# $param_match::  Set inital if command have no parameters or
+#             use partial if first n parameters match
+#             Default: undef
+#
 define keepalived::vrrp::track_process (
   String[1] $proc_name,
-  Optional[Integer[0]] $weight   = undef,
-  Integer[0] $quorum             = 1,
-  Optional[Integer[0]] $delay    = undef,
-  Boolean $full_command           = false
+  Optional[Integer[0]] $weight           = undef,
```

Comment:
These extra parameters also need to be documented and have tests such as what was done with `param_match`

Reply:
This is only a realignment. There are no extra parameters except _param_match_ in this change.